megatron/model/adapter.py

```diff
@@ -88,7 +115,7 @@ def forward(self, x: TensorType["b", "s", "d"], *attn_args, **attn_kwargs):
             attn_outputs[0],
             attn_outputs[1:],
         )  # output_attn: a, present, (attentions)
-        hidden_states = attn_output + (self.adapter(x) * self.adapter_scale)
+        hidden_states = attn_output + (self.adapt(x) * self.adapter_scale)
```

Comment:
- Not the same as whats done in non parallel version?
- This results in:
result = (1+scale)attn_output + scaleadapter_branch(attn_output)

Doesnt look like that should be the case?